### DISCUSSION TITLE
Zellij does not update when terminal size changes

### DISCUSSION THREAD

**Discussion by ldelossa** [zellij-org/zellij#3625] [Q&A]
Hello,

I connect to a VM over ssh and start  zellij session. 

When I move this terminal to another monitor that's bigger Zellij never updates. 

This leaves Zellij panes small and do not use up the extended space. 

Unfortunately this remains when I rejoin the session. 

![image](https://github.com/user-attachments/assets/7a6ba12a-e66d-4d13-b615-030b87e70c8d)

**Comment by giyany** [comment]
when several users connect from different machines, the smallest screen width/height is used for the session.
You can disconnect a user with Ctrl-o to open the session manager and Ctrl-x to disconnects other users.

**Reply by ldelossa** [reply]
Okay, I do not believe I had more then one user connected but let me test.

**Reply by Lockszmith-GH** [reply]
In your screenshot, the tab on the left (cilium) shows two colors, those are multiple connections - you might not have intended it, but somehow you got 2 concurrent connections.
One the of the connection is forcing the small size, like giyany stated.
If you move to another tab, you'll see that one of the colors remain on cilium - that's the session that is troubling you.

**Reply by ldelossa** [reply]
Thank you!! I didn't know this.
